spec/ruby-lint/constant_loader_spec.rb

```diff
@@ -2,8 +2,13 @@
 
 describe RubyLint::ConstantLoader do
   before do
+    @registry    = RubyLint::Definition::Registry.new
+    # hack: get the core definitions that hardcode the global registry
+    @registry.instance_variable_set(:@registered,
+                                    RubyLint.registry.registered.dup)
```

Comment:
No, I don't want tests doing things such as setting/getting instance variables directly. If the purpose is to stub out whatever registry is used you should use RSpec stubs for this.